.yarnrc.yml

```diff
@@ -1,12 +1,12 @@
+compressionLevel: mixed
+
+enableGlobalCache: false
+
 nodeLinker: node-modules
 # allow NPM_TOKEN env to be set, but provide '' as fallback. This allows CI to publish packages, but not require env var.
 # See https://yarnpkg.com/configuration/yarnrc
 npmAuthToken: "${NPM_TOKEN-}"
 
 npmPublishAccess: public
 
-plugins:
-  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
-    spec: "@yarnpkg/plugin-interactive-tools"
```

Comment:
This plugin is automatically included in Yarn 4. See: https://www.npmjs.com/package/@yarnpkg/plugin-interactive-tools#install

Reply:
Sweet! I think that's for things like `yarn upgrade-interactive`